src/main/java/com/models/Order.java

```diff
@@ -0,0 +1,47 @@
+package com.models;
```

Comment:
```suggestion
package com.model;
```
package names should be singular